react/components/EXPERIMENTAL_Table/DataTable/RowTree.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react'
+
+import NodeRow from './NodeRow'
+import useTableContext from '../hooks/useTableContext'
+
+const RowTree: FC = () => {
```

Comment:
Don't you think it might be strange to have these simple components extracted? How are they different from a regular `items.map`? It's not that clear.

Reply:
Just stopped doing that